task_manager/tests/conftest.py

```diff
@@ -1,7 +1,15 @@
 import pytest
 from django.test import Client
+from task_manager.users.models import ServiceUser
 
 
 @pytest.fixture
 def client() -> Client:
     return Client()
+
+
+@pytest.fixture()
```

Comment:
[optional] not sure if you need to call it if you don't intend to provide any extra parameters to the fixture